File: The Cooler Dilemma 1.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y,m;
        scanf("%d%d%d",&x,&y,&m);
        if(x*m<y)
        printf("YES\n");
        else
        printf("NO\n");
    }
}

File: The Cooler Dilemma 1.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            int m=sc.nextInt();
            if(x*m<y)
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}